dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -97,10 +98,11 @@ BeginDialog Dialog1, 0, 0, 401, 70, "Verifications Needed"
 EndDialog
 
 'Runs the dialog to allow workers to sign and to list verifications needed
-Do 
+Do
     Do
     	Dialog Dialog1
-    	Cancel_without_confirmation
+    	cancel_without_confirmation
+		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
```

Comment:
Same as the NEW HIRE's: Err_msg isn't set in the do...loop. No msgbox code to give the worker_signature error message.